### PR TITLE
[l10n] Improve Hungarian (hu-HU) locale

### PR DESCRIPTION
**Pull request by PetakCC**

### CODE REVIEW

File: packages/grid/x-data-grid/src/locales/huHU.ts [hunk 1]
```diff
@@ -26,16 +26,16 @@ const huHUGrid: Partial<GridLocaleText> = {
   toolbarFiltersTooltipActive: (count) => `${count} aktív szűrő`,
 
   // Quick filter toolbar field
-  // toolbarQuickFilterPlaceholder: 'Search…',
-  // toolbarQuickFilterLabel: 'Search',
-  // toolbarQuickFilterDeleteIconLabel: 'Clear',
+  toolbarQuickFilterPlaceholder: 'Keresés…',
+  toolbarQuickFilterLabel: 'Keresls',
```
Comment (brncsk) [inline packages/grid/x-data-grid/src/locales/huHU.ts:30]: Typo

Comment (LukasTy) [reply packages/grid/x-data-grid/src/locales/huHU.ts:30]:
```suggestion
  toolbarQuickFilterLabel: 'Keresés',
```

**Review by LukasTy — COMMENTED** [review]
Thank you for the contribution! 🙏 

Could you please run `yarn l10n` and commit the changes? 🤔

**Comment by CleverCC** [comment]
> Thank you for the contribution! 🙏
> 
> Could you please run `yarn l10n` and commit the changes? 🤔

Ofc, done :)

**Review by oszbence — APPROVED** [review]

**Review by oszbence — APPROVED** [review]

**Review by LukasTy — APPROVED** [review]

**Pull request merged**